src/pkg/sdk/index.ts

```diff
@@ -1,38 +1,64 @@
 // Copyright © 2022 Ory Corp
 // SPDX-License-Identifier: Apache-2.0
-import { Configuration, FrontendApi, OAuth2Api } from "@ory/client"
+import {
+  Configuration,
+  FrontendApi,
+  IdentityApi,
+  OAuth2Api,
+  PermissionApi,
+} from "@ory/client"
 
 const baseUrlInternal =
   process.env.ORY_SDK_URL || "https://playground.projects.oryapis.com"
 
-const apiBaseFrontendUrlInternal =
-  process.env.KRATOS_PUBLIC_URL || baseUrlInternal
+const apiBaseFrontendUrl = process.env.KRATOS_PUBLIC_URL || baseUrlInternal
 
-const apiBaseOauth2UrlInternal = process.env.HYDRA_PUBLIC_URL || baseUrlInternal
+const apiBaseIdentityUrl = process.env.KRATOS_ADMIN_URL || baseUrlInternal
 
-export const apiBaseUrl =
-  process.env.KRATOS_BROWSER_URL || apiBaseFrontendUrlInternal
+const apiBaseOauth2UrlInternal = process.env.HYDRA_ADMIN_URL || baseUrlInternal
+
+const apiBasePermissionsUrlInternal =
+  process.env.KETO_READ_URL || baseUrlInternal
+
+const kratosBrowserUrl = process.env.KRATOS_BROWSER_URL || apiBaseFrontendUrl
 
 const hydraBaseOptions: any = {}
 
 if (process.env.MOCK_TLS_TERMINATION) {
   hydraBaseOptions.headers = { "X-Forwarded-Proto": "https" }
 }
 
+const baseConfig = new Configuration({
+  basePath: apiBaseFrontendUrl,
+})
+
+const frontendConfig = new Configuration({
+  ...baseConfig,
+})
+
+const identityConfig = new Configuration({
+  ...baseConfig,
+  basePath: apiBaseIdentityUrl,
+})
+
+const permissionsConfig = new Configuration({
+  ...baseConfig,
+  basePath: apiBasePermissionsUrlInternal,
+})
```

Comment:
We don't Keto inside the Account Experience anywhere. I would rather suggest we remove it then.